internal/jimm/offer/authorizer_test.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2025 Canonical.
+
+package offer_test
+
+import (
+	"context"
+	"database/sql"
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/frankban/quicktest/qtsuite"
+	"github.com/google/uuid"
+	"github.com/juju/names/v5"
+	gc "gopkg.in/check.v1"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/jimm/offer"
+	"github.com/canonical/jimm/v3/internal/testutils/jimmtest"
+)
+
+type offerAuthorizerSuite struct {
+	offerAuthorizer *offer.OfferAuthorizer
+
+	offerUUID string
+}
+
+const offerAuthEnv = `clouds:
```

Comment:
Can you simplify this env? I.e. it doesn't look like `fred@canonical.com` is used.